docsrc/imap/reference/admin/sop/deleting.rst

```diff
@@ -15,7 +15,7 @@ Message context
 
 .. glossary::
 
-    Delete
+    Deleted
```

Comment:
These terms exist twice because they mean different things in different contexts, and both meanings need to be defined.  Changing the word silences the warning, but makes the documentation worse.